libs/cli/langchain_cli/namespaces/template.py

```diff
@@ -115,9 +122,13 @@ def serve(
     host_str = host if host is not None else "127.0.0.1"
 
     script = (
-        "langchain_cli.dev_scripts:create_demo_server"
-        if not configurable
-        else "langchain_cli.dev_scripts:create_demo_server_configurable"
+        "langchain_cli.dev_scripts:create_demo_server_chat_playground"
```

Comment:
typo... this should be create_demo_server_chat to match the naem of the function in dev_scripts.py